src/one/nio/os/Cpus.java

```diff
@@ -48,7 +48,7 @@ private static BitSet cpus(String rangeFile) {
             return cpus;
         } catch (IOException e) {
             if (log.isDebugEnabled()) {
```

Comment:
Now `log.isDebugEnabled()` check is unnecessary.

Reply:
Fixed (also in a few other places).